index.bs

```diff
@@ -690,6 +706,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
 
     <li><i><dfn>Exposure Compensation</dfn></i> is a numeric camera setting that adjusts the exposure level from the current value used by the implementation.  This value can be used to bias the exposure level enabled by auto-exposure, and usually is a symmetric range around 0 EV (the no-compensation value).</li>
 
+    <li><i><dfn>Exposure Time</dfn></i> is a numeric camera setting that controls the length of time during which light is allowed to fall on the photosensitive device. This value is used in manual exposure mode to control exposure. The value is in 100 microsecond units. That is, a value of 1.0 means an exposure time of 1/10000th of a second and a value of 10000.0 means an exposure time of 1 second.</li>
```

Comment:
Please s/manual exposure mode/`{{MeteringMode/manual}} {{exposureMode}}`/ for better autolinking.
Same in l.699.